openApiDocs/beta/Teams.Chats.yml

```diff
@@ -2260,20 +2281,23 @@ components:
         name: string
         thumbnailUrl: string
     microsoft.graph.chatMessageMention:
-      title: chatMessageMention
-      type: object
-      properties:
-        id:
-          maximum: 2147483647
-          minimum: -2147483648
-          type: integer
-          format: int32
-          nullable: true
-        mentionText:
-          type: string
-          nullable: true
-        mentioned:
-          $ref: '#/components/schemas/microsoft.graph.identitySet'
+      allOf:
+        - $ref: '#/components/schemas/microsoft.graph.entity'
```

Comment:
This $ref is will cause build errors as described https://github.com/microsoftgraph/msgraph-beta-sdk-dotnet/pull/110.

Reply:
I've opened https://github.com/microsoftgraph/msgraph-metadata/issues/22 to track this issue.